### PR TITLE
chore: add Open Notificaties to local Docker Compose setup

### PR DESCRIPTION
**Pull request by edgarvonk**
Added Open Notificaties to local Docker Compose setup. 

Solves PZ-4641

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -453,6 +483,89 @@ services:
       - "18083:8080"
     profiles: [ "itest" ]
 
+  rabbitmq:
+    image: rabbitmq:3.12.2-alpine
+    profiles: [ "opennotificaties" ]
```
Comment (hsiliev) [inline docker-compose.yaml:488]:
```suggestion
    healthcheck:
      test: rabbitmq-diagnostics -q ping
      interval: 30s
      timeout: 30s
      retries: 3
    profiles: [ "opennotificaties" ]
```

**Review by hsiliev — APPROVED** [review]

**Comment by edgarvonk** [comment]
A few notes:

- Unfortunately I had to use host.docker.internal instead of the local Docker network host names to connect OpenZaak and OpenNotificaties (and vice versa). Not sure why.. This is ok because we only want to run OpenNotificaties locally anyway and not in our itests.
- I added an option o to our start-docker-compose.sh script to also run OpenNotificaties (and enable it in OpenZaak). It is not enabled by default (but maybe we could change that) also because we don't want it in our itests.
- I still get some errors in the OpenNotificaties and OpenZaak Celery logs. Not sure why. It does seem to work ok though.
- Both OpenZaak and OpenNotificaties require an additional Celery Docker container which does the actual sending of the notifications from OpenZaak to OpenNotificaties and the callbacks from OpenNotificaties to ZAC.
- Since we don't use this setup in our itest it also means that it is not tested automatically. So this setup is a bit brittle. Same as with our metrics setup btw.

File: docker-compose.yaml (hunk 1 above)
Comment (edgarvonk) [reply docker-compose.yaml:488]: Nice, thanks. Will add. Not sure if it is really needed but looks give it is good to have. I copied this setup from the OpenNotificaties Docker Compose file where they also don't have a healthcheck for RabbitMQ..

File: scripts/docker-compose/imports/opennotificaties-database/database/1-setup-applicatie.sql [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+INSERT INTO datamodel_filtergroup (abonnement_id, kanaal_id) VALUES((SELECT ID FROM datamodel_abonnement where client_id = 'opennotificaties'), (SELECT ID FROM datamodel_kanaal where naam = 'zaaktypen'));
+INSERT INTO datamodel_filtergroup (abonnement_id, kanaal_id) VALUES((SELECT ID FROM datamodel_abonnement where client_id = 'opennotificaties'), (SELECT ID FROM datamodel_kanaal where naam = 'zaken'));
+INSERT INTO datamodel_filtergroup (abonnement_id, kanaal_id) VALUES((SELECT ID FROM datamodel_abonnement where client_id = 'opennotificaties'), (SELECT ID FROM datamodel_kanaal where naam = 'documenten'));
+INSERT INTO datamodel_filtergroup (abonnement_id, kanaal_id) VALUES((SELECT ID FROM datamodel_abonnement where client_id = 'opennotificaties'), (SELECT ID FROM datamodel_kanaal where naam = 'besluittypen'));
+INSERT INTO datamodel_filtergroup (abonnement_id, kanaal_id) VALUES((SELECT ID FROM datamodel_abonnement where client_id = 'opennotificaties'), (SELECT ID FROM datamodel_kanaal where naam = 'besluiten'));
```
Comment (infonl-marcel) [inline scripts/docker-compose/imports/opennotificaties-database/database/1-setup-applicatie.sql:36]: Taken needed here as well?

**Review by infonl-marcel — APPROVED** [review]

**Pull request merged**